.github/workflows/manual-release.yml

```diff
@@ -48,48 +48,69 @@ jobs:
+#    with:
+#      APP_NAME: ${{ needs.set-vars.outputs.app-name }}
+#      RUNNER_LABEL: ${{ needs.start-runner.outputs.label }}
+#      INFRASTRUCTURE_FOLDER: "infrastructure/aws"
+#      PUBLIC_INFRA_DEPLOYMENT: true
+#      ENVIRONMENT: ${{ github.event.inputs.env }}
+#      INFRA_CONFIG_REPO: ${{ needs.set-vars.outputs.infra-config-repo }}
+#    secrets:
+#      AWS_GITHUBRUNNER_PAT: ${{ secrets.AWS_GITHUBRUNNER_PAT}}
+#      AWS_GITHUBRUNNER_PAT_USER: ${{ secrets.AWS_GITHUBRUNNER_PAT_USER }}
+#      AWS_REGION: ${{ secrets.AWS_REGION }}
+#      AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+
   release-terraform:
+    runs-on: ${{ needs.start-runner.outputs.label }}
     needs:
       - set-vars
       - start-runner
-    uses: i-dot-ai/i-dot-ai-core-github-actions/.github/workflows/apply-terraform.yml@main
-    with:
-      APP_NAME: ${{ needs.set-vars.outputs.app-name }}
-      RUNNER_LABEL: ${{ needs.start-runner.outputs.label }}
-      INFRASTRUCTURE_FOLDER: "infrastructure/aws"
-      PUBLIC_INFRA_DEPLOYMENT: true
-      ENVIRONMENT: ${{ github.event.inputs.env }}
-      INFRA_CONFIG_REPO: ${{ needs.set-vars.outputs.infra-config-repo }}
-    secrets:
-      AWS_GITHUBRUNNER_PAT: ${{ secrets.AWS_GITHUBRUNNER_PAT}}
-      AWS_GITHUBRUNNER_PAT_USER: ${{ secrets.AWS_GITHUBRUNNER_PAT_USER }}
-      AWS_REGION: ${{ secrets.AWS_REGION }}
-      AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+    steps:
+      - name: Checkout redbox-copilot repo
+        uses: actions/checkout@v4
+        with:
+          path: redbox-copilot
 
-#  release:
-#    runs-on: ${{ needs.start-runner.outputs.label }}
-#    needs:
-#      - set-vars
-#      - start-runner
-#    steps:
-#      - uses: actions/checkout@v3
-#
-#      - name: Set up Python 3.11
-#        uses: actions/setup-python@v3
-#        with:
-#          python-version: "3.11"
-#
-#      - name: Make release
-#        run: |
-#          make tf_auto_deploy env=${{ github.event.inputs.env }} IMAGE_TAG=${{ github.event.inputs.image_tag }}
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+
+      - name: Checkout i-dot-ai-core-terraform repo
+        uses: actions/checkout@v4
+        with:
+          repository: i-dot-ai/i-ai-core-infrastructure
+          ref: refs/heads/main
+          token: ${{ secrets.AWS_GITHUBRUNNER_PAT }}
+          path: i-ai-core-infrastructure
+
+      - name: Checkout i-dot-ai-redbox-config repo
+        uses: actions/checkout@v4
+        with:
+          repository: i-dot-ai/redbox-copilot-infra-config
+          ref: refs/heads/main
+          token: ${{ secrets.AWS_GITHUBRUNNER_PAT }}
+          path: redbox-copilot-infra-config
+
+      - name: Make release
+        working-directory: redbox-copilot
+        run: |
+          make tf_init
+          make tf_auto_deploy env=${{ github.event.inputs.env }} IMAGE_TAG=${{ github.event.inputs.image_tag }}
```

Comment:
Is `github.event.inputs.env` variable will set as `dev` for manual deployment?

Reply:
yeah I think we'll do it that way, afaik dev is the only one we want to be done manually. I've left it configurable for testing in case Lewis needs to use dev, I can just swap to use preprod instead

Reply:
Make sense.